var/spack/repos/builtin/packages/py-wxpython/package.py

```diff
@@ -11,13 +11,15 @@ class PyWxpython(PythonPackage):
     homepage = "https://www.wxpython.org/"
     pypi = "wxPython/wxPython-4.0.6.tar.gz"
 
+    version("4.2.2", sha256="5dbcb0650f67fdc2c5965795a255ffaa3d7b09fb149aa8da2d0d9aa44e38e2ba")
     version("4.1.1", sha256="00e5e3180ac7f2852f342ad341d57c44e7e4326de0b550b9a5c4a8361b6c3528")
     version("4.0.6", sha256="35cc8ae9dd5246e2c9861bb796026bbcb9fb083e4d49650f776622171ecdab37")
 
     depends_on("c", type="build")  # generated
     depends_on("cxx", type="build")  # generated
 
     depends_on("wxwidgets +gui")
+    depends_on("wxwidgets@3.2.6 +gui", when="@4.2.2")
```

Comment:
We try to keep the dependencies correct, even if not all versions are included. Is it possible this should be:
```suggestion
    depends_on("wxwidgets@3.2.6: +gui", when="@4.2:")
```
i.e. all versions 4.2 or higher (including 4.2.2) must have at least wxwidgets version 3.2.6 or higher? 

Or is wxpython only making a statement about whatever version is bundled, and not including any required minimum version? In that case, I'd use something like
```suggestion
    depends_on("wxwidgets@3.2.6:3.2 +gui", when="@4.2.2")
```
since bug fixes in semver should not affect the interface (possibly even `wxwidgets@3.2.6:3` if minor releases are not expected to break compatibility).

Reply:
I'm not sure what the minimum version of `wxwidgets` is for any given version of `wxpython` and can't find any documentation that states this.  I just setup this rule since `wxpython@4.2.2` was bundled with `wxwidgets@3.2.6`.  I can potentially try loosening this up, or at least going off semver rules as you suggest.

Reply:
If this is the  bundled version and there is no additional information, then let's keep as is and hope that there are no conflicts. If there are, we can relax after more study.